src/modules/map/components/footer/MapFooterBackgroundSelector.vue

```diff
@@ -20,17 +20,14 @@
                 :data-cy="`background-selector-${background.getID()}`"
                 @click="selectBackground(background, $event)"
             >
-                <div class="bg-selector-button-label">
+                <div class="bg-selector-button-label text-wrap">
```

Comment:
We might want to modify a little bit the no background here.
See how it looks with your PR :
![image](https://github.com/geoadmin/web-mapviewer/assets/11540521/549e4051-d6a7-4173-b73b-b05cb3b431ed)

And on "prod"
![image](https://github.com/geoadmin/web-mapviewer/assets/11540521/71a7a8b7-4086-4fd9-8fe0-970006b3a027)

I don't mind the wrap, but it changes the visual "logic". It looks a bit wrong to me this way. We could keep the wrap but maybe add a border around the empty rectangle.

Reply:
@pakb The wrap is needed for the grey map in french otherwise we don't see anything. It is with the wrap like on the old viewer.
To be honest I don't like at all the style of the bg selector and the code is horribly static with lots of CSS magics, so you cannot add dynamically an additional background for example. So I propose to let this like this for now and to re-think and eventually rework the bg selector ui overall.